docs/data/material/pagesApi.js

```diff
@@ -68,7 +68,6 @@ module.exports = [
   { pathname: '/material-ui/api/list-item-secondary-action' },
   { pathname: '/material-ui/api/list-item-text' },
   { pathname: '/material-ui/api/list-subheader' },
-  { pathname: '/material-ui/api/loading-button' },
```

Comment:
We should put a redirection in place so people land on `/material-ui/api/button`

Reply:
I'm not sure that's best. Wouldn't it be misleading? People expect deprecated LoadingButton but get directed to the Button API.

Reply:
Do you have an example of how it was handled in the past when a component was removed?

Reply:
If we want to keep the `LoadingButton` API page around, we should change the strategy and undo the changes to the `LoadingButton` files in @mui/lab, and mark it as deprecated instead of replacing its implementation with `Button`.

cc @mnajdova

Reply:
I would recommend following what's done with the [Hidden](https://mui.com/material-ui/api/hidden/) component.

Reply:
The `Hidden` component was already in Material UI so it made sense to mark it as deprecated first and then remove it later on (which I guess where the removal should have been done in v6). But the `LoadingButton` is a Lab component where MUI Lab is already an experimental package ([About the lab](https://deploy-preview-42987--material-ui.netlify.app/material-ui/about-the-lab/)). [The docs](https://mui.com/material-ui/react-button/#experimental-apis) also inform it as a experimental API.

For example, `Alert` and `AlertTitle` were moved from Lab to core in #22651.

Given this, we should remove LoadingButton from Lab and add the loading feature to Material UI Button.

Reply:
I'm fine with either approach, but let's make sure the approach we pick is the one we used most recently.